src/components/account/AccountInfo.tsx

```diff
@@ -0,0 +1,44 @@
+import {faPen} from '@fortawesome/free-solid-svg-icons';
+import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
+import {useNavigation} from '@react-navigation/native';
+import {Image, Text, TouchableOpacity, View} from 'react-native';
+
+const AccountInfo: React.FC = () => {
+  const navigation = useNavigation();
+
+  return (
+    <View className="flex-row p-5">
+      <TouchableOpacity
+        className="relative mr-5 h-16"
+        onPress={() => navigation.navigate('ChangeInfo')}>
```

Comment:
hạn chế dùng inline function
```suggestion
const  onChangeInfo = () => {
navigation.navigate('ChangeInfo')
};
        onPress={onChangeInfo}>
```